logic/src/commonMain/kotlin/com/wire/kalium/logic/configuration/UserConfigRepository.kt

```diff
@@ -120,6 +121,8 @@ interface UserConfigRepository {
     suspend fun deleteLegalHoldRequest(): Either<StorageFailure, Unit>
     suspend fun setLegalHoldChangeNotified(isNotified: Boolean): Either<StorageFailure, Unit>
     suspend fun observeLegalHoldChangeNotified(): Flow<Either<StorageFailure, Boolean>>
+    suspend fun setCRLExpirationTime(domain: String, url: String, timestamp: ULong)
```

Comment:
```suggestion
    suspend fun setCRLExpirationTime(crlUrl: String, url: String, timestamp: ULong)
```

Reply:
There is a chance that we have different crls per domain, so I wanted that to be accurate.